src/main.c

```diff
@@ -26,6 +26,8 @@ in Windows that produces packet capture events) to pcapng format
 #include <tdh.h>
 #include <strsafe.h>
 #include <pcapng.h>
+#include <winsock2.h>
```

Comment:
Put these before pcapng.h to maintain the rough "general-to-specific" ordering of headers.